src/edit.js

```diff
@@ -36,46 +48,66 @@ export default function Edit( { attributes, setAttributes } ) {
 	useEffect( () => {
 		async function getRelatedPosts() {
 			const postId = window.location.href.match( /post=([\d]+)/ )[ 1 ];
-			const response = await window.fetch( `/wp-json/wp/v2/posts?search=${ countries[ countryCode ] }&exclude=${ postId }` );
+			const response = await window.fetch(
+				`/wp-json/wp/v2/posts?search=${ countries[ countryCode ] }&exclude=${ postId }`
+			);
```

Comment:
In the context of block editor, we should better use `@wordpress/api-fetch` (for built-in middlewares like a one adding a nonce) but ideally related posts should be fetched using WordPress data layer. Moreover, since we only need a few fields and not the entire post object, we should filter the fields we want to receive using the `_field` property.

```js
	const { fetchedRelatedPosts, fetchingRelatedPosts } = useSelect(
		( select ) => {
			const getEntityRecordsArgs = [
				'postType',
				'post',
				{
					search: countries[ countryCode ],
					_fields: [ 'id', 'link', 'title', 'excerpt' ],
					exclude: postId,
				},
			];

			return {
				fetchedRelatedPosts: select( 'core' ).getEntityRecords(
					...getEntityRecordsArgs
				),
				fetchingRelatedPosts: select( 'core' ).getIsResolving(
					'getEntityRecords',
					getEntityRecordsArgs
				),
			};
		},
		[ countryCode, postId ]
	);
```